### PR TITLE
Fixes for ABU-686. Would also suggest all values for things such as t…

### PR DESCRIPTION
**Pull request by danielghost**
…he state object are taken from the model rather than the assessmentConfig. This will be reviewed separately.

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1

File: js/adapt-assessmentArticleModel.js [hunk 1]
```diff
@@ -38,12 +38,12 @@ define([
...
 			var attemptsLeft;
 			switch (assessmentConfig._attempts) {
-			case "infinite": case 0: case undefined: case -1: case null:
-				 attemptsLeft = "infinite";
-				break;
-			default:
-				attemptsLeft = assessmentConfig._attempts;
-				break;
+				case "infinite": case 0: case undefined: case -1: case null:
+					 attemptsLeft = "infinite";
```
Comment (oliverfoster) [inline js/adapt-assessmentArticleModel.js]: suggest normalising the value of _attempts on the model to "infinite" as well.

File: js/adapt-assessmentArticleModel.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ define([
 				'_attemptInProgress': false, 
 				'_numberOfQuestionsAnswered': 0,
 				'_lastAttemptScoreAsPercent': 0,
+				"_attempts": attemptsLeft,
```
Comment (oliverfoster) [inline js/adapt-assessmentArticleModel.js]: gotcha

**Comment by moloko** [comment]
+1

**Comment by taylortom** [comment]
+1

**Pull request merged**